patches/youch@4.1.0-beta.10.patch

```diff
@@ -0,0 +1,66 @@
+diff --git a/build/chunk-4XB2BYKC.js b/build/chunk-4XB2BYKC.js
+index 6e6e490115bc4b107b53834ceac3d7d19cdbe3fe..37373d734d034caf45e28c1b165e5c176f901137 100644
+--- a/build/chunk-4XB2BYKC.js
++++ b/build/chunk-4XB2BYKC.js
+@@ -34,10 +34,10 @@ var ErrorStackSource = class extends BaseComponent {
+    */
+   async toHTML(props) {
+     const frame = props.frame;
+-    if (frame.type === "native" || !frame.source || !frame.fileName) {
++    if (!frame.source || !frame.fileName) {
```

Comment:
Youch currently skip displaying the script source if it is an internal module. But we do want it to show up if `frame.source` is defined. I will put up a PR to youch  later.